packages/shorebird_cli/lib/src/code_push_client_wrapper.dart

```diff
@@ -412,7 +412,7 @@ Please create a release using "shorebird release" and try again.
     required Iterable<Arch> architectures,
     String? flavor,
   }) async {
-    final createArtifactProgress = logger.progress('Creating artifacts');
+    final createArtifactProgress = logger.progress('Uploading artifacts');
```

Comment:
```suggestion
    final uploadArtifactProgress = logger.progress('Uploading artifacts');
```
can we rename the variable for consistency?

Reply:
Actually it might be better to split this into two parts (creating and uploading) for more transparency and accuracy? wdyt?

Reply:
In many cases the "creating" is the calling of the API, which implicitly does the "uploading".